### PR TITLE
Add release notes and compatibility info for LC 2.5 release

### PR DESCRIPTION
**Pull request by sonotley**
## What Changed?

Added release notes for LC 2.5

I have added PG15 and PGD5 to the table of supported software.

I have also removed BDR where it is used as a product name for what is
now called EDB Postgres Distributed. Uses of bdr in variables or
database objects are still correct so have been left.

### CODE REVIEW

**Comment by drothery-edb** [comment]
@sonotley, jfyi @ebgitelman has opened another [PR](https://github.com/EnterpriseDB/docs/pull/4029) where she is doing a copy edit and rebranding to PGD.

**Review by wind39 — CHANGES_REQUESTED** [review]
Looks good, only 1 minor typo.

**Comment by sonotley** [comment]
> @sonotley, jfyi @ebgitelman has opened another [PR](https://github.com/EnterpriseDB/docs/pull/4029) where she is doing a copy edit and rebranding to PGD. @dougcalobrisi had requested our assistance with the rebranding.

Ok, this PR needs to go out for the release on Tuesday so we can resolve conflicts after that. I knew Doug reached out but didn;t know work had started.

File: product_docs/docs/livecompare/2/bdr_support.mdx [hunk 1]
```diff
@@ -1,39 +1,40 @@
...
-navTitle: BDR support
-title: BDR Support
+navTitle: PGD support
+title: EDB Postgres Distributed Support
 originalFilePath: bdr_support.md
 
 ---
 
-LiveCompare can be used against BDR nodes, as well as non-BDR nodes.
+LiveCompare can be used against EDB Postgres Distributed (PGD, formerly know as BDR)
```
Comment (sonotley) [inline product_docs/docs/livecompare/2/bdr_support.mdx:8]:
```suggestion
LiveCompare can be used against EDB Postgres Distributed (PGD, formerly known as BDR)
```

**Review by nidhibhammar — DISMISSED** [review]

File: product_docs/docs/livecompare/2/bdr_support.mdx [hunk 2]
```diff
@@ -112,39 +113,39 @@ replication_sets = set_name = 'bdrgroup'
 Settings `node_name` and `replication_sets` are supported for the following
 technologies:
 
--   BDR 1, 2, 3 and 4;
+-   PGD 1, 2, 3, 4 and 5;
```
Comment (drothery-edb) [inline product_docs/docs/livecompare/2/bdr_support.mdx:116]: Not sure we want to list v1 and v2, but we can deal with that later

File: product_docs/docs/livecompare/2/supported_technologies.mdx [hunk 3]
```diff
@@ -34,11 +34,11 @@ in which context they can be used in LiveCompare.
...
 | ------------------------------ | ------------------------------- | --------------------------- |
-| PostgreSQL                     | 10, 11, 12, 13 and 14 | Data and/or Output          |
-| EDB PostgreSQL Extended        | 10, 11, 12, 13 and 14      | Data and/or Output          |
-| EDB PostgreSQL Advanced (EPAS) | 11, 12, 13 and 14               | Data and/or Output          |
+| PostgreSQL                     | 10, 11, 12, 13, 14 and 15 | Data and/or Output          |
+| EDB PostgreSQL Extended        | 10, 11, 12, 13, 14 and 15     | Data and/or Output          |
+| EDB PostgreSQL Advanced (EPAS) | 11, 12, 13, 14 and 15               | Data and/or Output          |
 | pglogical                      | 2 and 3                         | Initial, Data and/or Output |
-| BDR                            | 1, 2, 3 and 4                   | Initial, Data and/or Output |
+| EDB Postgres Distributed       | 1, 2, 3, 4 and 5                   | Initial, Data and/or Output |
```
Comment (drothery-edb) [inline product_docs/docs/livecompare/2/supported_technologies.mdx:41]: again, I'm not sure we want to include 1 and 2 - we have no docs for those versions

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**